[2022-08-06 19:38:14,213] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-01T00:00:00+00:00 [queued]>
[2022-08-06 19:38:14,226] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-01T00:00:00+00:00 [queued]>
[2022-08-06 19:38:14,226] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:14,226] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:38:14,226] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:14,243] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-03-01 00:00:00+00:00
[2022-08-06 19:38:14,250] {standard_task_runner.py:52} INFO - Started process 2129 to run task
[2022-08-06 19:38:14,253] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-03-01T00:00:00+00:00', '--job-id', '646', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpu72gj6jf', '--error-file', '/tmp/tmpavrni7nq']
[2022-08-06 19:38:14,253] {standard_task_runner.py:80} INFO - Job 646: Subtask second_task
[2022-08-06 19:38:14,324] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-01T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:38:14,419] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-03-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-01T00:00:00+00:00
[2022-08-06 19:38:14,420] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:38:14,421] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:38:14,435] {subprocess.py:85} INFO - Output:
[2022-08-06 19:38:14,437] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:38:14,437] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:38:14,531] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220301T000000, start_date=20220806T193814, end_date=20220806T193814
[2022-08-06 19:38:14,584] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:38:14,624] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:52:05,704] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-01T00:00:00+00:00 [queued]>
[2022-08-06 21:52:05,718] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-01T00:00:00+00:00 [queued]>
[2022-08-06 21:52:05,718] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:05,718] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:52:05,719] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:05,741] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-03-01 00:00:00+00:00
[2022-08-06 21:52:05,748] {standard_task_runner.py:52} INFO - Started process 2317 to run task
[2022-08-06 21:52:05,751] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-03-01T00:00:00+00:00', '--job-id', '648', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpbixsjcfh', '--error-file', '/tmp/tmpmg_v2upj']
[2022-08-06 21:52:05,751] {standard_task_runner.py:80} INFO - Job 648: Subtask second_task
[2022-08-06 21:52:05,835] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-01T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:52:05,933] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-03-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-01T00:00:00+00:00
[2022-08-06 21:52:05,934] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:52:05,935] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:52:05,951] {subprocess.py:85} INFO - Output:
[2022-08-06 21:52:05,953] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:52:05,953] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:52:06,046] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220301T000000, start_date=20220806T215205, end_date=20220806T215206
[2022-08-06 21:52:06,083] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:52:06,130] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:57:29,241] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-01T00:00:00+00:00 [queued]>
[2022-08-06 21:57:29,255] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-01T00:00:00+00:00 [queued]>
[2022-08-06 21:57:29,255] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:57:29,255] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:57:29,255] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:57:29,272] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-03-01 00:00:00+00:00
[2022-08-06 21:57:29,280] {standard_task_runner.py:52} INFO - Started process 2105 to run task
[2022-08-06 21:57:29,284] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-03-01T00:00:00+00:00', '--job-id', '647', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmprmo5af2t', '--error-file', '/tmp/tmp0ajzir7c']
[2022-08-06 21:57:29,284] {standard_task_runner.py:80} INFO - Job 647: Subtask second_task
[2022-08-06 21:57:29,363] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-01T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:57:29,538] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-03-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-01T00:00:00+00:00
[2022-08-06 21:57:29,539] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:57:29,540] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:57:29,562] {subprocess.py:85} INFO - Output:
[2022-08-06 21:57:29,565] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:57:29,612] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:57:29,670] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220301T000000, start_date=20220806T215729, end_date=20220806T215729
[2022-08-06 21:57:29,697] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:57:29,738] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:12:32,489] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-01T00:00:00+00:00 [queued]>
[2022-08-06 22:12:32,508] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-01T00:00:00+00:00 [queued]>
[2022-08-06 22:12:32,508] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:12:32,509] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:12:32,509] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:12:32,529] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-03-01 00:00:00+00:00
[2022-08-06 22:12:32,538] {standard_task_runner.py:52} INFO - Started process 2122 to run task
[2022-08-06 22:12:32,543] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-03-01T00:00:00+00:00', '--job-id', '650', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpn96f3un6', '--error-file', '/tmp/tmp0m0guodu']
[2022-08-06 22:12:32,543] {standard_task_runner.py:80} INFO - Job 650: Subtask second_task
[2022-08-06 22:12:32,662] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-01T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:12:32,774] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-03-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-01T00:00:00+00:00
[2022-08-06 22:12:32,775] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:12:32,775] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:12:32,792] {subprocess.py:85} INFO - Output:
[2022-08-06 22:12:32,793] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:12:32,794] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:12:32,833] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220301T000000, start_date=20220806T221232, end_date=20220806T221232
[2022-08-06 22:12:32,880] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:12:32,931] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:23:53,321] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-01T00:00:00+00:00 [queued]>
[2022-08-06 22:23:53,340] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-01T00:00:00+00:00 [queued]>
[2022-08-06 22:23:53,340] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:23:53,340] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:23:53,340] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:23:53,360] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-03-01 00:00:00+00:00
[2022-08-06 22:23:53,367] {standard_task_runner.py:52} INFO - Started process 2178 to run task
[2022-08-06 22:23:53,370] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-03-01T00:00:00+00:00', '--job-id', '651', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpp_w_6fzw', '--error-file', '/tmp/tmpdwkh92sq']
[2022-08-06 22:23:53,370] {standard_task_runner.py:80} INFO - Job 651: Subtask second_task
[2022-08-06 22:23:53,437] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-01T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:23:53,517] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-03-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-01T00:00:00+00:00
[2022-08-06 22:23:53,518] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:23:53,518] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:23:53,531] {subprocess.py:85} INFO - Output:
[2022-08-06 22:23:53,533] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:23:53,533] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:23:53,564] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220301T000000, start_date=20220806T222353, end_date=20220806T222353
[2022-08-06 22:23:53,581] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:23:53,611] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
